net/url/test.cc

```diff
@@ -35,37 +35,37 @@ TEST(URLTest, SmokeTest) {
   u = URL("www.google.com");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/", u.path);
-  EXPECT_EQ("http", u.protocol);
+  EXPECT_EQ("http", u.scheme);
   EXPECT_EQ(80, u.port);
 
   u = URL("www.google.com/test");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/test", u.path);
-  EXPECT_EQ("http", u.protocol);
+  EXPECT_EQ("http", u.scheme);
   EXPECT_EQ(80, u.port);
 
   u = URL("www.google.com:8080");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/", u.path);
-  EXPECT_EQ("http", u.protocol);
+  EXPECT_EQ("http", u.scheme);
   EXPECT_EQ(8080, u.port);
 
   u = URL("meh://www.google.com:27960");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/", u.path);
-  EXPECT_EQ("meh", u.protocol);
+  EXPECT_EQ("meh", u.scheme);
   EXPECT_EQ(27960, u.port);
 
   u = URL("meh://www.google.com:27960/bazinga");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/bazinga", u.path);
-  EXPECT_EQ("meh", u.protocol);
+  EXPECT_EQ("meh", u.scheme);
   EXPECT_EQ(27960, u.port);
 
   u = URL("localhost:/test");
```

Comment:
I meant without the path ("test") but with the slash. Either `URL("localhost/")` or `URL("localhost:/")`, the colon is tested in your test, the path with the only slash isn't.